openreview/venue_request/process/revisionProcess.py

```diff
@@ -6,6 +6,9 @@ def process(client, note, invitation):
     conference = openreview.helpers.get_conference(client, note.forum, SUPPORT_GROUP)
     forum_note = client.get_note(note.forum)
 
+    note.writers = []
```

Comment:
instead of doing this, why don't we set the writers empty in each Stage invitation?